pages/common/ansible-vault.md

```diff
@@ -0,0 +1,28 @@
+# ansible-vault
+
+> Encrypts & decrypts values, data structures and files within Ansible projects.
+> More information: <https://docs.ansible.com/ansible/latest/user_guide/vault.html#id17>.
+
+- Create a new encrypted vault file with a prompt for a password:
```

Comment:
Perhaps "Create a new encrypted vault file; user will be prompted for a password:" sonuds better ? "with a prompt" reads a bit weird to me. And this also makes it same as the `psql` page.